### ISSUE TITLE
The application-route-mixin should look up the baseURL from the ember-cli enviroment configuration

### ISSUE THREAD

**Issue by luxferresum** [simplabs/ember-simple-auth#537]
Hi

ember-simple-auth is checking the routers `rootURL` while setting up the configuration, but the `baseURL` of ember-cli seems to work without setting the `rootURL`.

I think this should work out of the box.

Currently I use this action in my application route:

    sessionInvalidationSucceeded: function() {
        if(!Ember.testing) {
            window.location.replace(this.container.lookup('router:main').get('location.baseURL'));
        }
    }

**Comment by marcoow** [comment]
Doesnt Ember CLI set the router's `rootUrl` according to what's configured as `baseUrl`? I was under the impression that it was actually necessary for the router to work correctly.

**Comment by heathharrelson** [comment]
I ran into this very issue a few days ago. Ember CLI doesn't seem to include `baseURL` in the router's `rootURL`. The `rootURL` gets the default value of '/' unless you set it explicitly.

I ended up doing the following in my application route to get the behavior I wanted:

```javascript
    sessionInvalidationSucceeded: function () {
      if (!Ember.testing) {
        var baseURL = config.baseURL !== '/' ? config.baseURL : '';
        var rootURL = this.get('router').get('rootURL');

        window.location.replace(baseURL + rootURL);
      }
    }
```

I actually didn't know `rootURL` existed until I tried to change the logout behavior. There's no reference to it in the `config/environment.js` or `app/router.js` files generated by Ember CLI (just tried with 0.2.7).

**Comment by luxferresum** [comment]
Okay, rootURL seems to be an internal, its moved to `this.container.lookup('router:main').get('location.concreteImplementation.baseURL')` in Ember 1.12.
I think it somehow works by setting the `<base href="/my/base/">`tag in the `<head>` and then the router can handle it.

**Comment by marcoow** [comment]
This will be fixed by #522 then. for now, @heathharrelson's fix should work.
